nameko/standalone/events.py

```diff
@@ -48,5 +51,23 @@ def dispatch(service_name, event_type, event_data):
                     routing_key=routing_key,
                     retry=retry,
                     retry_policy=retry_policy,
+                    mandatory=mandatory,
                     **kwargs)
+
+                if mandatory:
+                    if not use_confirms:
+                        warnings.warn(
```

Comment:
a warning is fine for now, but i think it would be better to choose either error or nothing